ui/webui/test/check-storage

```diff
@@ -408,17 +433,7 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
         disk = "/dev/vda"
         dev = "vda"
         btrfsname = "btrfstest"
-        m.execute(f"""
-        sgdisk --zap-all {disk}
-        sgdisk --new=0:0:+1MiB -t 0:ef02 {disk}
-        sgdisk --new=0:0:+1GiB {disk}
-        sgdisk --new=0:0:+10GiB {disk}
-        sgdisk --new=0:0:0 {disk}
-        mkfs.ext4 {disk}2
-        mkfs.xfs -f {disk}3
-        mkfs.btrfs -f -L {btrfsname} {disk}4
-        """)
-
+        self.partition_disk(disk,  [("1MiB", "biosboot"), ("1GB", "ext4"), ("10GiB", "xfs", "-f"), ("", "btrfs", "-f", "-L", btrfsname)])
```

Comment:
You can safely always use the -f/-F flags. The inconsistency in the tests is not delibarate. take care `-f` is for btrfs and xfs. ext4 has `-F`

Reply:
So just to be clear - don't make the extra mkfs flags parameter - this is not needed. Always force the mkfs.

Reply:
done